site/content/docs/5.1/components/orange-navbar.md

```diff
@@ -1,49 +1,458 @@
+        <br>
+        Title Line 2
+      </h1>
+    </div>
+
+    <!-- Burger menu (visible on small screens) -->
+    ...
+
+    <!-- Navbar with links -->
+    ...
+
+    <!-- Navbar with action icons -->
+    ...
+  </div>
+</nav>
+```
+
+### With search box
+
+You can add a search input into your Global header.
+
+<div class="bd-example p-0">
+  <header>
+    {{< orange-global-headers id="global-header-5" mode="search" demo=true aria_label="Global navigation - With search box example">}}
+    {{< /orange-global-headers >}}
+  </header>
+</div>
+
+```html
+<nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation - With search box example">
+  <div class="container-xxl">
+
+    <!-- Orange brand logo -->
+    ...
+
+    <!-- Burger menu (visible on small screens) -->
+    ...
+
+    <!-- Navbar with links -->
+    ...
+
+    <!-- Navbar with action icons -->
+    <form class="d-flex d-none d-lg-flex nav-item" role="search">
+      <!-- todo : reforge when .dark-input -->
+      <input class="form-control bg-transparent text-white border border-2 border-dark" type="search" placeholder="Search" aria-label="Search">
+    </form>
+    <div id="global-header-5.2" class="navbar-collapse collapse d-sm-flex global-header-5">
+      <ul class="navbar-nav flex-row">
+        <li class="nav-item d-lg-none">
+          <a href="#" class="nav-link nav-icon">
+            <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false" class="overflow-visible">
+              <use xlink:href="/docs/{{<param docs_version>}}/assets/img/boosted-sprite.svg#search" />
+            </svg>
+            <span class="visually-hidden">Search</span>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+```
+
+### Minimizing behavior
+
+When scrolling the page, for breakpoints from `md` to `xxl`, the Global header's height decreases when `.sticky-top` is used on `<header>`.
+
+You can see a full example of the Minimizing Orange Navbar behavior in action [here]({{<docsref "/examples/navbar-sticky">}}).
+
+The minimized global header will **always** use the small [Master logo]({{<docsref "/about/brand#orange-logo">}}).
+
+{{< callout warning >}}
+Be careful when using **`.fixed-top`**, the collapsed content should be escapable easily.
+{{< /callout >}}
+
+## Navigation under
+
+A Navigation under bar is a simple navigation bar `<nav>`, different than a Supra bar. It can be used as a last part of an Orange navbar whenever your header may have too many items to stand on one line.
+
+It should be visible only for the larger screens (`lg` to `xxl` breakpoints) using `.navbar-expand-*`.
+
+<div class="bd-example p-0">
+  <header>
+    {{< orange-global-headers id="global-header-6" mode="actions" title=true title_2=true labels=true navigation=true aria_label="Global navigation - Standard example with nav-under">}}
+    {{< /orange-global-headers >}}
+  </header>
+</div>
+
+```html
+<header>
```

Comment:
All the examples in the page could have the `<header>` surrounding tag in order for the users to not to forget it.